docs/components/saas/byok/cost-and-troubleshooting.md

```diff
@@ -0,0 +1,40 @@
+---
+id: cost-and-troubleshooting
+title: "Cost implications and troubleshooting"
```

Comment:
Hard to get context.
External Encryption Keys cost implications and troubelshooting